image/base/Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV RAILS_ENV production
 
 RUN echo 2.0.`date +%Y%m%d` > /VERSION
```

Comment:
Why it doesn't like that line? 🤔 

```
#6 [ 2/54] RUN echo 2.0.`date +%Y%m%d` > /VERSION
#6 0.278 standard_init_linux.go:228: exec user process caused: exec format error
#6 ERROR: executor failed running [/bin/sh -c echo 2.0.`date +%Y%m%d` > /VERSION]: exit code: 1

error: failed to solve: executor failed running [/bin/sh -c echo 2.0.`date +%Y%m%d` > /VERSION]: exit code: 1
```

Reply:
Oh that is expected :sweat_smile: 

Gotta setup the whole emulator before hand, still WIP